tests/testthat/test-snapshot-file.R

```diff
@@ -31,6 +31,29 @@ test_that("expect_snapshot_file works", {
   )
 })
 
+
+test_that("expect_snapshot_file works in a different directory", {
+  skip_if_not(getRversion() >= "3.6.0")
+  current_wd <- getwd()
+  path <- tempfile()
+  dir.create(path)
+
+  withr::defer({
+    setwd(current_wd)
+    unlink(path)
+  })
+
+  setwd(path)
```

Comment:
I think you can replace all this with `withr::local_tempdir()` 😄

Reply:
Good point!